flow/Stats.h

```diff
@@ -91,14 +91,28 @@ struct Counter : ICounter, NonCopyable {
 
 	Value getIntervalDelta() const { return interval_delta; }
 	Value getValue() const { return interval_start_value + interval_delta; }
-	double getRate() const;				// dValue / dt
-	double getRoughness() const;			// value deltas come in "clumps" of this many ( 1 = periodic, 2 = poisson, 10 = periodic clumps of 10 (nearly) simultaneous delta )
+
+	// dValue / dt
+	double getRate() const;
+
+	// Measures the clumpiness or dispersion of the counter.
+	// Computed as a normalized variance of the time between each incrementation of the value.
+	// A delta of N is treated as N distinct increments, with N-1 increments having time span 0.
+	// Normalization is performed by dividing each time sample by the mean time before taking variance.
+	//
+	// roughness = Variance(t/mean(T)) for time samples t in T
```

Comment:
This comment block about the roughness' definition is very useful!

Can you please define what `t` and `T` is in this comment block?
Maybe a simple and short example following this line can help understanding.

Reply:
This is just a restatement of the above paragraph, where I was attempting to provide full definitions. If there’s anything unclear, about that paragraph, though, let me know and I can try to reword it.